src/styles/blocks/_product-card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 0 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  margin-right: 49px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 46px;
+  }
+
+  &__img {
+    margin: 32px auto 36px;
+  }
+
+}
+
+.img {
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: var(--text-normal);
+  line-height: 1.5;
+  color: var(--text-color);
+}
+
+.card__id {
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: var(--text-small);
+  line-height: 1.4;
+  color: var(--secondary-color);
+}
+
+.card__feedback {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__comments {
+  font-size: var(--text-small);
+  line-height: 1.4;
+}
+
+.card__price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.price-title {
+  font-weight: 400;
+  font-size: var(--text-normal);
+  line-height: 1.5;
+  color: var(--secondary-color);
+}
+
+.price-value {
+  font-weight: 700;
+  font-size: var(--text-big);
+  line-height: 1.12;
+  color: var(--text-color);
+}
+
+.card__btn {
+  margin-bottom: 16px;
+  display: block;
+  width: 166px;
+  height: 40px;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  background-color: var(--blue-color);
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 40px;
+  color: white;
+  border: 1px solid transparent;
+  border-radius: 5px;
+}
+
+.card__btn:hover {
```

Comment:
[SASS] - Make use of SASS nesting - write pseudo-class, pseudo-element selectors inside general selector. As well as media queries.